module/programs/helper.js

```diff
@@ -6,15 +6,11 @@
  */
 
 // Dependencies
-const entitiesHelper = require(MODULES_BASE_PATH + '/entities/helper');
 const timeZoneDifference = process.env.TIMEZONE_DIFFRENECE_BETWEEN_LOCAL_TIME_AND_UTC;
 const validateEntity = process.env.VALIDATE_ENTITIES;
 const userService = require(ROOT_PATH + '/generics/services/users');
-const userRolesHelper = require(MODULES_BASE_PATH + '/userRoles/helper');
-const programUsersHelper = require(MODULES_BASE_PATH + '/programUsers/helper');
```

Comment:
@praveenKDass please confirm that these imports are not required

Reply:
not Required @VISHNUDAS-tunerlabs